mobile/library/cc/engine_builder.h

```diff
@@ -123,7 +123,8 @@ class XdsBuilder final {
 class EngineBuilder {
 public:
   EngineBuilder();
-  virtual ~EngineBuilder() {}
+  EngineBuilder(EngineBuilder&&) = default;
```

Comment:
why do we need this line?

Reply:
It's because the `EngineBuilder` is now a movable type because it has `std::unique_ptr` as its member.